trackpoint/src/module.c

```diff
@@ -133,6 +154,38 @@ static bool readByte()
     return false;
 }
 
+int16_t errorCode = 0;
+
+#define RQ_TO_SEND(AT)                  \
```

Comment:
I have made these macros in order to make adding new commands into the loop easier. (For testing purposes.)

Should I leave them here, or revert to the original?

Reply:
For me, it's harder to digest the code with the macros included, so I'd prefer not to have them.